modules/changePassword.php

```diff
@@ -0,0 +1,92 @@
+<?php
+if (!$user) {
+    header("location: index.php");
+} else {
+    $user_id = $user["id"];
+    $user_pw = $user["password"];
+}
+
+// define variable to check if success change passowrd
+if (isset($_GET['success'])) {
+    $isSuccess = $_GET['success'];
+} else {
+    $isSuccess = false;
+}
+
+// Process to change password
+if (!empty($_POST)) {
+
+    $curPass = md5($_POST['currentPassword']);
+    $newPass = md5($_POST['newPassword']);
+    $confPass = md5($_POST['confirmPassword']);
+
+    if ($newPass != $confPass ) {
+        unset($_POST);
+        header("location: index.php?m=changePassword&confMatch=0");
```

Comment:
You don't need to handle by this way, let put error to an array, and show them if there is error.